tests/garage/tf/policies/test_gaussian_mlp_policy_with_model.py

```diff
@@ -0,0 +1,181 @@
+            policy4.model.networks['default'].distribution)
+        objective2 = tf.reduce_mean(kl_diff_sym2)
+
+        kl2 = self.sess.run(
+            objective2,
+            feed_dict={
+                policy3.model.networks['default'].input: obs,
+                policy4.model.networks['default'].input: obs
+            })
+
+        self.assertAlmostEqual(kl1, kl2)
+
+        ############################################################
+        # log_likelihood_sym
+        # * Existing way *
+        log_prob_sym1 = policy1.distribution.log_likelihood_sym(
+            obs_ph, dist1_sym)
+        log_prob_func = tensor_utils.compile_function([obs_ph], log_prob_sym1)
+        log_prob1 = log_prob_func(obs)
+
+        # * New way *
+        # equvaient to distribution.log_likelihood_sym(X, dist)
+        log_prob_sym2 = policy3.model.networks[
+            'default'].distribution.log_prob(
+                policy3.model.networks['default'].input)
+        log_prob2 = self.sess.run(  # result
+            log_prob_sym2,
+            feed_dict={policy3.model.networks['default'].input: obs})
+
+        assert log_prob1 == log_prob2
+
+        ############################################################
+        # entropy_sym
+        # * Existing way *
+        entropy_sym1 = policy1.distribution.entropy_sym(
+            dist1_sym, name='entropy_sym1')
+        entropy_func = tensor_utils.compile_function([obs_ph], entropy_sym1)
+        entropy1 = entropy_func(obs)  # result
+
+        # * New way *
+        # equvaient to garage.tf.distribution.entropy_sym(dist)
+        entropy_sym2 = policy3.model.networks['default'].distribution.entropy()
+        entropy2 = self.sess.run(  # result
+            entropy_sym2,
+            feed_dict={policy3.model.networks['default'].input: obs})
+
+        assert entropy1 == entropy2
+
+        ############################################################
+        # likelihood_ratio_sym
+        # * Existing way *
+        likelihood_ratio_sym1 = policy1.distribution.likelihood_ratio_sym(
+            obs_ph, dist1_sym, dist2_sym, name='li_ratio_sym1')
+        likelihood_ratio_func = tensor_utils.compile_function(
+            [obs_ph], likelihood_ratio_sym1)
+        likelihood_ratio1 = likelihood_ratio_func(obs)
+
+        # * New way *
+        # tf.distributions seems doesn't have this available
+        # maybe do it ourselves
+        with tf.name_scope('li_ratio_sym2'):
+            log_prob_diff = policy4.model.networks[
+                'default'].distribution.log_prob(
+                    obs_ph, name='log_prob_obs') - policy3.model.networks[
+                        'default'].distribution.log_prob(obs_ph)
+
+            likelihood_ratio_sym2 = tf.exp(log_prob_diff)
+            likelihood_ratio2 = self.sess.run(  # result
+                likelihood_ratio_sym2,
+                feed_dict={
+                    policy3.model.networks['default'].input: obs,
```

Comment:
Is there a more concise way to call the inputs and outputs? It is exhausting to get the output in a simple policy by `policy.model.networks['default'].output`. 
This seems trivial if all the reviewers think it is ok then you can ignore this.